### DISCUSSION TITLE
Cannot comment on ticket outstanding

### DISCUSSION THREAD

**Discussion by ghost** [community/community#49430] [New to GitHub]
I have resorted to speaking here for I cannot open a new ticket as it is accusing me of receiving "a few new tickets from my account recently." I've only ever filed one two days ago.

My account is flagged due to my email, and I changed it as per the ticket instructions. The ticket is marked as pending; however, I cannot remark that I have made the desired changes. GitHub says that "You cannot comment on this ticket. You do not have an email in the verified domain or that has already been CCd." My new email is a verified Gmail account.

I hope the people behind the flagging system reconsider their choice to delegate this task to support. This is a great hassle for simply putting up a blog on GitHub Pages.

**Comment by juanker-cso** [comment]
Did you solve it? I'm having the exact same issue right now.

**Comment by Favio2424** [comment]
same issue
